src/test/agent/test_instance.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from mock import patch, MagicMock
+from spacel.agent.instance import InstanceManager
+from spacel.model import AgentManifest
+
+
+class TestInstanceManager(unittest.TestCase):
+    def setUp(self):
+        self.instance = InstanceManager()
+        self.manifest = AgentManifest({
+            'healthcheck': {
+                'endpoint': 'test',
+                'virtual_host': 'spacel',
+                'protocol': 'https'
+            }
+        })
+
+    @patch('spacel.agent.instance.InstanceManager.check_health')
+    def test_health(self, mock_check_health):
+        mock_check_health.return_value = True
```

Comment:
Can't this just be `self.instance._check_health = MagicMock(return_value=True)`. Patch feels heavy